Library/Homebrew/env_config.rb

```diff
@@ -474,5 +474,12 @@ def cask_opts_quarantine?
     def cask_opts_require_sha?
       cask_opts.include?("--require-sha")
     end
+
+    sig { returns(T::Array[String]) }
+    def pip_index_urls
```

Comment:
This should be a `HOMEBREW_PIP_INDEX_URL` variable and added to the hash above instead of being taken from the environment unconditionally.